### DISCUSSION TITLE
Deploy/dev/Kubernetes/nginx-public-ingress

### DISCUSSION THREAD

**Discussion by dikshauat** [Sunbird-Ed/Community#181] [General Discussion]
Hi @keshavprasad, @shounak, I am getting the following error in cloudshell for nginx-public-ingress on kubernets pods

kubectl logs nginx-public-ingress-7t94j  -n dev

2022/09/19 16:35:36 [emerg] 1#1: invalid host in upstream "10.128.0.27:8080/" in /etc/nginx/nginx.conf:68
nginx: [emerg] invalid host in upstream "10.128.0.27:8080/" in /etc/nginx/nginx.conf:68

Please help me resolving the issue

**Comment by beepdot** [comment]
This job should be deployed at the end. After all other Kubernetes deployment jobs succeed. Please drop a note back here if you face issues even after deploying all other services.

**Reply by dikshauat** [reply]
Hi @keshavprasadms ,

Thanks for your response. We did the same thing but still getting the error. Also, when we are trying to access keycloak on 8080, it says 404-not found. Kindly help me resolving the issue.

Regards

**Reply by beepdot** [reply]
What does the keycloak log say? Please check the errors in keycloak vm at `/var/log/keycloak.out.log`.
Also please share me the `keycloak` upstream configuration from the configmap named `nginx-conf`. It looks like this
```
        upstream keycloak {
            server 11.2.1.12;
            keepalive 1000;
        }
```

**Reply by dikshauat** [reply]
Thanks for the help @keshavprasadms . keycloak is up now. When we are running nginx-public-ingress we are still getting this error on pods:

invalid host in upstream "10.128.0.27:8080/"

We tried locating nginx-conf file on keycloak server but were not able to find it. Please help us with the path . Also, help us with the solution of the error

**Reply by dikshauat** [reply]
Thanks @keshavprasadms , we are now able to access Sunbird'page on our cloud infra. Please help us with the issue we are facing with search of deploy
